### ISSUE TITLE
Crash: -[UIKeyboardTaskQueue waitUntilAllTasksAreFinished] may only be called from the main thread.

### ISSUE THREAD

**Issue by jleandroperez** [wordpress-mobile/WordPress-iOS#5144]
```
Thread : Crashed: com.twitter.crashlytics.ios.exception
0  WordPress                      0x1004d69a0 CLSProcessRecordAllThreads + 141976
1  WordPress                      0x1004d6e74 CLSProcessRecordAllThreads + 143212
2  WordPress                      0x1004c23a8 CLSHandler + 58528
3  WordPress                      0x1004d5688 __CLSExceptionRecord_block_invoke + 137088
4  libdispatch.dylib              0x180fb147c _dispatch_client_callout + 16
5  libdispatch.dylib              0x180fbc728 _dispatch_barrier_sync_f_invoke + 100
6  WordPress                      0x1004d5204 CLSExceptionRecord + 135932
7  WordPress                      0x1004d503c CLSExceptionRecordNSException + 135476
8  WordPress                      0x1004d4c74 CLSTerminateHandler() + 134508
9  libc++abi.dylib                0x180bbef44 std::__terminate(void (*)()) + 16
10 libc++abi.dylib                0x180bbe85c __cxxabiv1::exception_cleanup_func(_Unwind_Reason_Code, _Unwind_Exception*) + 134
11 libobjc.A.dylib                0x180bcc094 _objc_exception_destructor(void*) + 330
12 CoreFoundation                 0x181566d08 +[NSException raise:format:] + 106
13 Foundation                     0x181eec124 -[NSAssertionHandler handleFailureInMethod:object:file:lineNumber:description:] + 112
14 UIKit                          0x1866ddbb0 -[UIKeyboardTaskQueue waitUntilAllTasksAreFinished] + 176
15 UIKit                          0x1866de438 -[UIKeyboardImpl setDelegate:force:] + 1084
16 UIKit                          0x1866d7eb0 -[UIPeripheralHost(UIKitInternal) _reloadInputViewsForResponder:] + 1180
17 UIKit                          0x186d494bc -[UIPeripheralHost(UIKitInternal) _preserveInputViewsWithId:animated:reset:] + 488
18 UIKit                          0x186a31228 -[UIViewController _presentViewController:modalSourceViewController:presentationController:animationController:interactionController:completion:] + 1168
19 UIKit                          0x186a32d64 -[UIViewController _presentViewController:withAnimationController:completion:] + 4644
20 UIKit                          0x186a357a0 -[UIViewController _performCoordinatedPresentOrDismiss:animated:] + 472
21 UIKit                          0x1867ab2b4 -[UIViewController presentViewController:animated:completion:] + 184
22 WordPress                      0x1002765b0 UIAlertController.presentFromRootViewController() -> () (UIAlertController+Helpers.swift:40)
23 WordPress                      0x100276670 @objc UIAlertController.presentFromRootViewController() -> () (UIAlertController+Helpers.swift)
24 WordPress                      0x1001325e0 +[WPError showAlertWithTitle:message:withSupportButton:okPressedBlock:] (WPError.m:170)
25 WordPress                      0x100132254 +[WPError showAlertWithTitle:message:] (WPError.m:132)
26 WordPress                      0x100099b3c __44-[WPPostViewController addDeviceMediaAsset:]_block_invoke (WPPostViewController.m:1814)
27 WordPress                      0x100348f00 partial apply for thunk (PHAsset+Exporters.swift)
28 WordPress                      0x100349c18 partial apply for thunk (PHAsset+Exporters.swift)
29 WordPress                      0x10034b9ec PHAsset.(exportVideoToURL(PHAsset) -> (NSURL, targetUTI : String, maximumResolution : CGSize, stripGeoLocation : Bool, successHandler : (resultingSize : CGSize) -> (), errorHandler : (error : NSError) -> ()) -> ()).(closure #1) (PHAsset+Exporters.swift:128)
30 WordPress                      0x10034a488 partial apply for PHAsset.(exportVideoToURL(PHAsset) -> (NSURL, targetUTI : String, maximumResolution : CGSize, stripGeoLocation : Bool, successHandler : (resultingSize : CGSize) -> (), errorHandler : (error : NSError) -> ()) -> ()).(closure #1) (PHAsset+Exporters.swift)
31 WordPress                      0x10034be70 thunk (PHAsset+Exporters.swift)
32 Photos                         0x18b2736dc __82-[PHImageManager requestExportSessionForVideo:options:exportPreset:resultHandler:]_block_invoke + 516
33 Photos                         0x18b26f460 __100-[PHImageManager _requestAsynchronousVideoURLForAsset:chainedToMasterRequest:options:resultHandler:]_block_invoke_4 + 960
34 Photos                         0x18b270510 __100-[PHImageManager _requestAsynchronousVideoURLForAsset:chainedToMasterRequest:options:resultHandler:]_block_invoke_21537 + 76
35 AssetsLibraryServices          0x18a197b14 __116-[PLGatekeeperClient requestVideoURLForAsset:format:networkAccessAllowed:streamingAllowed:trackCPLDownload:handler:]_block_invoke + 512
36 libxpc.dylib                   0x1811fcb4c _xpc_connection_reply_callout + 28
37 libxpc.dylib                   0x1811fcaf0 _xpc_connection_call_reply + 40
38 libdispatch.dylib              0x180fb147c _dispatch_client_callout + 16
39 libdispatch.dylib              0x180fbf914 _dispatch_root_queue_drain + 2140
40 libdispatch.dylib              0x180fbf0b0 _dispatch_worker_thread3 + 112
41 libsystem_pthread.dylib        0x1811c9470 _pthread_wqthread + 1092
42 libsystem_pthread.dylib        0x1811c9020 start_wqthread + 4
```

**Comment by SergioEstevao** [comment]
This issue is related to this one: https://github.com/wordpress-mobile/WordPress-iOS/issues/5043 and was solved by https://github.com/wordpress-mobile/WordPress-iOS/pull/5061 and already merged in 6.1

**Comment by jleandroperez** [comment]
Thanks for checking @SergioEstevao!

Closing + nuking the branch.
